npm/vue/cypress/support/commands.js

```diff
@@ -1,6 +1,6 @@
 /// <reference types="cypress" />
 import { mount } from '@cypress/vue'
 
-Cypress.Commands.add('mount', (comp) => {
+Cypress.Commands.overwrite('mount', (fn, comp) => {
```

Comment:
@cowboy wanted to get your thoughts here - the `overwrite` is kind of silly for the `mount` / `hover` commands, since the point of `overwrite` is to replace the method but have the original around if you want to call it. We could alternatively make it so that `mount` & `hover` are special cased so you can use `Cypress.Commands.add('mount'` without it throwing.

Reply:
@tgriesser it would be _awesome_ if `mount` & `hover` were special cased so people could use `Cypress.Commands.add()` without it throwing. Just as long as `Cypress.Commands.overwrite()` also still worked.